spec/tests/yaml/swiftnav/sbp/system/test_MsgStatusJournal.yaml

```diff
@@ -0,0 +1,62 @@
+description: Unit tests for swiftnav.sbp.system MsgStatusJournal
+generated_on: '2022-04-28 09:56:19.891285'
+package: sbp.system
+tests:
+- msg:
+    fields:
+      reporting_system: 1
+      sbp_version: 1025
+      total_status_reports: 100
+      sequence_descriptor: 16
```

Comment:
Sequence descriptor is combination of index and count:

0x0001 0000 -- means sequence count of 1, item index 0